lib/streamlit/credentials.py

```diff
@@ -29,6 +30,10 @@
 
 LOGGER = get_logger(__name__)
 
+# WT_SESSION is a Windows Terminal specific environment variable. If it exists,
+# we are on the latest Windows Terminal that supports emojis
+SHOW_EMOJIS = platform.system() != "Windows" or os.environ.get("WT_SESSION")
```

Comment:
it's currently only used for `EMAIL_PROMPT` but thought it'd be nice to have for other messages